src/client/components/section-header/section-header.component.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './section-header.style.css';
+
+export default function SectionHeader({ children, onClick, tabIndex }) {
+  return (
+    <div
+      className="section-header"
+      role="button"
+      onClick={onClick}
+      onKeyDown={onClick}
```

Comment:
It's an accessibility thing, eslint doesn't let me not have both: 
> Visible, non-interactive elements with click handlers must have at least one keyboard listener.eslint(jsx-a11y/click-events-have-key-events)

➡️ For reference, 1st paragraph here as well as the link in it: https://stackoverflow.com/a/54612218/7547895

I guess I could have created it using a `button` instead of a `div`. It's not too late for that if there would be less confusion...